<!DOCTYPE html>
<html>
  <body>
    <div id="selectedinfo">
    Read files placed here will contribute to a single assembly.
    </div>
    <div id="pairinfo">
<b><h3>Paired read library</h3></b>
<!--<p>Paired-end sequencing has advantages over the use of single reads. Paired-end reads give added structure to the genome assembly problem which can be useful when dealing with ambiguous or low complexity regions.</p>-->
<p><b>Read File 1 &amp; 2</b></br>
Many paired read libraries are given as file pairs, with each file containing half of each read pair. Paired read files are expected to be sorted such that each read in a pair occurs in the same Nth position as its mate in their respective files. These files are specified as READ FILE 1 and READ FILE 2. For a given file pair, the selection of which file is READ 1 and which is READ 2 does not matter.</p>
<b><h4>Advanced Parameters</h4></b>
<p><b>File 1 Interleaved</b></br>
Some paired libaries are available in a single file where each read in a pair occurs in succession. To specify such a file set this parameter to 'True'.</p>
<p><b>Mean Insert Size</b></br>
This refers to the mean insert size between paired reads. If you have this information you may provide it. If not the assembly algorithm will make an attempt to determine this value.</p>
<p><b>Std. Insert Size</b></br>
This refers to the standard deviation of the insert size between paired reads. If you have this information you may provide it. If not the assembly algorithm will make an attempt to determine this value.</p>
<p><b>Mate Paired</b></br>
Defines the orientation of read pairs. Setting Mate Paired to true indicates that the sequencing direction of the two reads in each pair is outward facing.</p>
<p><b>Platform</b></br>
The sequencing platform used for each library.</br>
“infer”: Infer sequencing platform from read files; “illumina”: Illumina short reads; “pacbio”: PacBio long reads; “nanopore”: MinION long reads
    </div>
    <div id="parameterinfo">
       <h3>Parameters</h3>
       <h4>Benchmark Contigs</h4>
       <p>This parameter can be used to specify a FASTA contigs file to evaluate the assembly against.</p>
       <h4>Output Folder</h4>
       <p>The workspace folder where results will be placed.</p>
       <h4>Output Name</h4>
       <p>Name used to uniquely identify results.</p>
      <h4>Assembly Strategy</h4>
        <p><b>auto</b></p>
        <p>&nbsp;&nbsp;For short reads:</p>
        <ol>
        <li>Runs BayesHammer on reads</li>
        <li>Assembles with Velvet, IDBA and SPAdes</li>
        <li>Sorts assemblies by ARAST quality score</li>
        </ol>
        <p>&nbsp;&nbsp;For long reads (PacBio or Nanopore):</p>
        <ol>
        <li>Assembles with MiniASM</li>
        </ol>
        <p><b>fast</b></p>
        <ol>
        <li>Assembles with MEGAHIT and Velvet.</li>
        <li>Results are sorted by ARAST quality score.</li>
        </ol>
        <p><b>full_spades</b></p>
        <ol>
        <li>Runs BayesHammer on reads</li>
        <li>Assembles with SPAdes.</li>
        </ol>
        <p><b>kiki</b></p>
        <ol>
        <li>Runs the Kiki assembler</li>
        </ol>
        <p><b>miseq</b></p>
        <ol>
        <li>Runs Velvet with hash length 35.</li>
        <li>Runs BayesHammer on reads and assembles with SPAdes with k up to 99.</li>
        <li>Results are sorted by ARAST quality score.</li>
        <li>Works for Illumina MiSeq reads.</li>
        </ol>
        <p><b>plasmid</b></p>
        <ol>
        <li>Runs BayesHammer on reads and assembles with plasmidSPAdes.</li>
        </ol>
        <p><b>smart</b></p>
        <p>&nbsp;&nbsp;For short reads:</p>
        <ol>
        <li>Runs BayesHammer on reads, Kmergenie to choose hash-length for Velvet</li>
        <li>Assembles with Velvet, IDBA and SPAdes</li>
        <li>Sorts assemblies by ALE score</li>
        <li>Merges the two best assemblies with GAM-NGS</li>
        </ol>
        <p>&nbsp;&nbsp;For long reads (PacBio or Nanopore):</p>
        <ol>
        <li>Assembles with MiniASM</li>
        </ol>
    </div>
    <div class="apphelp" id="pipelineinfo">
      <h4>Minimal output contig length</h4>
      Filter out short contigs in final assembly
      <h4>Minimal output contig coverage</h4>
      Filter out contigs with low read depth in final assembly
      <h4>Assembly Pipeline</h4>

      The pipeline parameter is an advanced way to customize the
      assembly workflow by mixing and matching a variety of
      modules. Each modules works at one of the three stages of the
      pipeline: preprocessing, assembly, and post-processing. In
      general, you can compose a pipeline by concating one or more
      preprocessing modules, one assembler, and optionally one
      postprocessor.

      </br>
      <h4>Example 1:</h4>
      <pre>   tagdust velvet</pre>
      This pipeline will simply run tagdust to remove adapter
      sequences in the reads and then assemble them with velvet.
      Note: quotes should not be used around the two modules as
      they have special meaning in pipeline syntax.

      </br>
      <h4>Example 2:</h4>
      <pre>   a6</pre>
      You can also invoke an assembler that we have not included in
      our curated strategies. In this case, A6 is an assembler with
      its built-in preprocessing and postprocessing steps.

      </br>
      <h4>Example 3:</h4>
      <pre>   "tagdust none" "megahit velvet" sspace</pre>
      You can use quotes to specify alternative modules you would
      like to try at each step. This example will launch
      a cartesian combination of four parallel pipelines:
      tagdust+megahit+sspace, tagdust+velvet+sspace, megahit+sspace, velvet+sspace.

      </br>
      <h4>Note:</h4>
      The pipeline parameter overrides the assembly strategy parameter.
      Not all modules combine well.

      <h4><a href="ftp://ftp.patricbrc.org/workshop/arast/arast_modules.txt" target="_blank">List of modules supported</a></h4>
    </div>
  </body>
</html>
